app/routes/company/components/CompaniesPage.js

```diff
@@ -17,36 +21,47 @@ type Props = {
 };
 
 const CompanyItem = ({ company }: Company) => {
-  const [protocol, , domain] = company.website
-    ? company.website.split('/')
-    : ['', '', ''];
-  const websiteString = `${protocol}//${domain}`;
-
   return (
     <div className={styles.companyItem}>
       <div className={styles.companyItemContent}>
-        <Link to={`/companies/${company.id}`}>
-          <div className={styles.companyLogo}>
-            {<Image src={company.logo} />}
-          </div>
-          <div className={styles.companyItemTitle}>
-            <h2>{company.name}</h2>
-          </div>
-          <div className={styles.companyInfo}>
-            <span>
-              {company.joblistingCount > 0 &&
-                company.joblistingCount + ' jobbannonser'}
+        <div className={styles.companyLogoContainer}>
+          <Link to={`/companies/${company.id}`}>
+            <div className={styles.companyLogo}>
+              {<Image src={company.logo} />}
+            </div>
+          </Link>
+        </div>
+        <Flex className={styles.companyInfo}>
+          <Flex column>
+            <Icon name="md-briefcase" prefix="ion-" size={20} />
+            <span className={styles.briefcaseCount}>
+              {company.joblistingCount}
             </span>
-            <span>
-              {company.eventCount > 0 && company.eventCount + ' arrangementer'}
-            </span>
-          </div>
-        </Link>
-        {company.website && !company.website.includes('example.com') && (
-          <a href={company.website} rel="noopener noreferrer" target="_blank">
-            <div className={styles.website}>{websiteString}</div>
-          </a>
-        )}
+          </Flex>
+          {company.website && (
```

Comment:
This is nice! We should however have a URL validator on the website field for the Company model in the backend, even though we do a bit of validation on the frontend when editing companies. Nothing you can do about it here though.